.github/workflows/smart_proxy_plugin.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Foreman Proxy Plugin
+
+on:
+  workflow_call:
+    inputs:
+      foreman_proxy_version:
+        description: Foraman Proxy version to use
+        default: 'develop'
+        required: false
+        type: string
+      foreman_proxy_repository:
+        description: Foreman Proxy repository to use
+        default: 'theforeman/smart-proxy'
+        required: false
+        type: string
+
+jobs:
+  rubies:
+    name: Verify supported Ruby versions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Verify supported Ruby versions
+        uses: theforeman/gha-matrix-ruby-verifier@v0
+        with:
+          repository: ${{ inputs.foreman_proxy_repository }}
+          ref: ${{ inputs.foreman_proxy_version }}
+  test:
```

Comment:
Can we make it easy to respect `inputs.foreman_proxy_version` in `Gemfile`? Like set an env var that we can use. Or is there some other way?

Reply:
env var it is